corehq/apps/dropbox/models.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from django.db import models
+from django.contrib.auth.models import User
+
+from .exceptions import DropboxUploadAlreadyInProgress
+from .tasks import upload
+
+
+class DropboxUploadHelper(models.Model):
+    """
+    This class is for working with dropbox uploads. It saves metadata to the database, stores the progress of
+    the upload, and ensures multiple uploads of the same file does not occur.
```

Comment:
this probably just stems from confusion on how i can use `django-soil`. this seemed like a slightly different use case. may be best to offline this